clients/gtest/geqrf_gtest.cpp

```diff
@@ -82,7 +82,51 @@ TEST_P(geqrf_gtest, geqrf_gtest_double)
 
     Arguments arg = setup_geqrf_arguments(GetParam());
 
-    hipblasStatus_t status = testing_geqrf<double>(arg);
+    hipblasStatus_t status = testing_geqrf<double, double>(arg);
+
+    if(status != HIPBLAS_STATUS_SUCCESS)
+    {
+        if(arg.M < 0 || arg.N < 0 || arg.lda < arg.M)
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_INVALID_VALUE, status);
+        }
+        else
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_NOT_SUPPORTED, status); // for cuda
```

Comment:
I think this should be `EXPECT_EQ(HIPBLAS_STATUS_SUCCESS, status);` since this is supported in CUDA - basically it should auto fail if it isn't a success.

Reply:
My understanding is that the non-batched geqrf is not supported in cuBLAS. It only has the batched version.